tests/testthat/test-unreachable_code_linter.R

```diff
@@ -121,6 +121,22 @@ test_that("unreachable_code_linter ignores code after foo$stop(), which might be
   )
 })
 
+test_that("unreachable_code_linter ignores terminal nolint end comments", {
```

Comment:
Should an inline `# nolint` comment also be ignored?

My first instinct is it should still throw a lint (i.e., we only ignore range exclusions)... either way, a test would be good.

Reply:
Can add a test. Would the XPath even find them? IINM it checks via `@line` attributes.

Reply:
TBC i meant like

```
stop()
# nolint
```

(not sure if it's a relevant concern -- couldn't think of a case where such an inline `# nolint` would be needed -- but thought a test would be good & wanted to be sure I wasn't missing anything)

I don't think this should lint:

```
stop() # comment on stop
```

Reply:
Oh, that's what you mean. But that nolint will never be useful, so why not lint it?
Agree with your `stop() # comment`.